psychopy_eyetracker_sr_research/sr_research/eyelink/eyetracker.py

```diff
@@ -155,90 +160,90 @@ def __init__(self, *args, **kwargs):
+        self._eyelinkSetScreenPhysicalData()
+
+        # Set whether to run in mouse simulation mode or not.
+        simulation_mode = tracker_config.get('simulation_mode', False)
+        if simulation_mode is True:
+            self._eyelink.sendCommand('aux_mouse_simulation = YES')
+        else:
+            self._eyelink.sendCommand('aux_mouse_simulation = NO')
+
+        # Set that the EyeLink connected button box, button 5
+        # (the big button on most of supported gamepads), will initiate
+        # an accept fixation command.
+        self._eyelink.sendCommand("button_function 5 'accept_target_fixation'")
+
+        # Set the 'runtime' configuration section of the eyetracker
+        # settings, including eye to track, sample filtering level, etc.
+        self._setRuntimeSettings(self._runtime_settings)
+
+        # Calibration related settings
+        self.sendCalibrationSettingsCommands(self._eyelink, tracker_config.get('calibration'))
+
+        # Set up the file names / paths to be used for the native EyeLink EDF file.
+        EyeTracker._local_edf_dir = EXP_SCRIPT_DIRECTORY
+
+        # native data recording file
+        default_native_data_file_name = tracker_config.get('default_native_data_file_name', None)
+        if isinstance(default_native_data_file_name, str):
+            if default_native_data_file_name == "EXPFILE":
+                # If edf file name has been set to EXPFILE, use the datastore file name as the local
+                # edf file name, getting around the 8 char host name limit.
+                EyeTracker._local_edf_dir = os.path.join(EyeTracker._local_edf_dir, "data")
+                if self._iohub_server.dsfile:
+                    local_file_name = self._iohub_server.dsfile.fileName[:-5]
+                    EyeTracker._full_edf_name = local_file_name
+                    EyeTracker._host_edf_name = default_native_data_file_name
+                else:
+                    # If datastore file is not enabled, attempt to use the PsychoPy experiment filename
+                    psychopy_file_name = None
+                    for dev in self._iohub_server.devices:
+                        if dev.__class__.__name__ == 'Experiment':
+                            psychopy_file_name = dev.getConfiguration().get('filename')
+                    if psychopy_file_name:
+                        # make sure local_file_name is relative to _local_edf_dir
+                        EyeTracker._local_edf_dir = os.path.commonprefix([self._local_edf_dir, psychopy_file_name])
+                        local_file_name = os.path.relpath(psychopy_file_name, self._local_edf_dir)
+                        EyeTracker._full_edf_name = local_file_name
+                        EyeTracker._host_edf_name = default_native_data_file_name
+            else:
+                r = default_native_data_file_name.rfind('.')
+                if r > 0:
+                    if default_native_data_file_name[r:].lower() == 'edf':
```

Comment:
this line was also changed due to 'edf'.lower() doing nothing, now it lowers the data file name so 'EXP.EDF' would match